Makefile

```diff
@@ -7,9 +7,10 @@ ADMIN_CONSOLE_CHART_NAME = admin-console
 ADMIN_CONSOLE_CHART_VERSION = 1.109.12
 ADMIN_CONSOLE_IMAGE_OVERRIDE =
 ADMIN_CONSOLE_MIGRATIONS_IMAGE_OVERRIDE =
+ADMIN_CONSOLE_BINARY_VERSION_OVERRIDE =
```

Comment:
there's already a `KOTS_VERSION` and `KOTS_BINARY_URL_OVERRIDE`. can we use those instead?